src/js/core/game.js

```diff
@@ -329,7 +329,7 @@ async function writeGameStats(tx, results, att) {
         keys.unshift("gs"); // Also record starters, in addition to other stats
         keys.push("pm");
         for (let p = 0; p < results.team[t].player.length; p++) {
-            gameStats.teams[t].players[p] = {name: results.team[t].player[p].name, pos: results.team[t].player[p].pos};
+            gameStats.teams[t].players[p] = {name: `${results.team[t].player[p].firstName} ${results.team[t].player[p].lastName}`, pos: results.team[t].player[p].pos};
```

Comment:
On second thought, it really doesn't like that.

![screen shot 2016-06-09 at 5 34 25 pm](https://cloud.githubusercontent.com/assets/910907/15950946/86d2caf8-2e68-11e6-8805-0cfd7363250f.png)